### PR TITLE
Added Array and tuples

### PR DESCRIPTION
**Pull request by mani-chand**
#1464 issue PR

### CODE REVIEW

**Comment by mani-chand** [comment]
Hello @djmitche , Can we make this a quick review if possible.

**Comment by djmitche** [comment]
It's the end of my day on a long weekend, so I can't promise that. I'm not sure what the rush is, but it should be possible for you to proceed working on other PRs before this one is merged, right? Also, I'm waiting for review on #1815 :)

**Comment by mani-chand** [comment]
Firstly, Have a great joyful long weekend.I will be working on other PRs. I am not getting notifications when you tag me in issues. I will go through #1815 in weekend and will get ready with my PR. There is no rush it's just I am free. 

Have great day and weekend.

File: src/tuples-and-arrays/tuples-and-arrays.md
Comment (randomPoison) [inline src/tuples-and-arrays/tuples-and-arrays.md:30]: It looks like you removed this `<!-- mdbook-xgettext: skip -->` comment from this code block. Please restore the comment so that this code block is still skipped for localization.

File: src/tuples-and-arrays/arrays/arrays.md
Comment (randomPoison) [inline src/tuples-and-arrays/arrays/arrays.md:3]: This code block is missing the `<!-- mdbook-xgettext: skip -->` comment that it had on the original slide. Please add it back in so that this code block isn't included in the translations.

**Review by randomPoison — CHANGES_REQUESTED** [review]
Splitting tuples and arrays into separate slides seems reasonable to me. The current iteration leaves the two slides pretty bare at the moment, though, since there's just a brief code example for both. I think with the separated slides we could move the high level summary of each type into the slide from the speaker notes. Or we should at least have more information to put on the separated array and tuple slides if we're going to split the slide up.

**Comment by mani-chand** [comment]
I have did few changes extra. Check them . If you want me to add more content in slide . I will add to slides .

File: src/SUMMARY.md
Comment (mgeisler) [inline src/SUMMARY.md:50]: When deleting a page, you have to add an entry to the redirection table in `book.toml`. That way we avoid 404 pages when people visit the old URL.

In this case, you should redirect `tuples-and-arrays/tuples-and-arrays.html` to `tuples-and-arrays/tuples.html`:

```toml
[output.html.redirect]
"tuples-and-arrays/tuples-and-arrays.html" = "tuples.html"
```

The tricky part is that the path to the right of `=` is relative to the path on the left.

Comment (mani-chand) [reply src/SUMMARY.md:50]: Thanks, Noted, I will check and make it works.

Comment (mgeisler) [reply src/SUMMARY.md:50]: Thanks!

**Comment by mani-chand** [comment]
Hello @mgeisler , Can you check.

File: src/tuples-and-arrays/tuples.md
Comment (mgeisler) [inline src/tuples-and-arrays/tuples.md:2]:
```suggestion
minutes: 5
```

**Comment by mgeisler** [comment]
> Hello @mgeisler , Can you check.

I'll let @randomPoison do the review, she is teaching the class regularly and has the most insight here.

**Review by randomPoison — APPROVED** [review]
This looks good now! You just need to rebase onto `main` and fix the conflicts and this should be good to merge.

**Comment by mani-chand** [comment]
@randomPoison  , I am sorry , I think I messed it up , I will open another PR.

**Comment by deavid** [comment]
The spanish translation change looks good, however there are other *.po files with way too many changes. Seeing how the other parts of the PR are minimal, I assume these po files diff can be reduced.

**Comment by djmitche** [comment]
> The spanish translation change looks good, however there are other *.po files with way too many changes. Seeing how the other parts of the PR are minimal, I assume these po files diff can be reduced.

I think this was because the PR had the wrong base, but thanks for looking!

**Pull request closed without merging**
